staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -340,8 +340,9 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 			valueWatchers: make(map[string]watchersMap),
 		},
 		// TODO: Figure out the correct value for the buffer size.
-		incoming:              make(chan watchCacheEvent, 100),
-		dispatchTimeoutBudget: newTimeBudget(stopCh),
+		incoming: make(chan watchCacheEvent, 100),
+		// set a clock for budget calculations
```

Comment:
remove this comment - it's obvious from the code that you pass clock

Reply:
Done. 👍🏽